libtiledbsoma/src/utils/geometry/utils.h

```diff
@@ -0,0 +1,30 @@
+#ifndef TILEDBSOMA_GEOMETRY_UTILS_H
```

Comment:
It doesn't look like `utils.h` is being used in this PR. Let's hold off on adding this in until it's in use.